examples/jaeger/src/main/java/io/opentelemetry/example/jaeger/ExampleConfiguration.java

```diff
@@ -26,18 +24,14 @@
   /**
    * Initialize an OpenTelemetry SDK with a Jaeger exporter and a SimpleSpanProcessor.
    *
-   * @param jaegerHost The host of your Jaeger instance.
-   * @param jaegerPort the port of your Jaeger instance.
+   * @param jaegerEndpoint The endpoint of your Jaeger instance.
    * @return A ready-to-use {@link OpenTelemetry} instance.
    */
-  static OpenTelemetry initOpenTelemetry(String jaegerHost, int jaegerPort) {
-    // Create a channel towards Jaeger end point
-    ManagedChannel jaegerChannel =
```

Comment:
Random aside - I was hoping most people weren't using `setChannel` in real apps but with it in our examples, now I'm less sure 😅